scripts/bridge-setup/bridge-setup.go

```diff
@@ -29,58 +29,67 @@ func main() {
 	privateKey := flag.String("privateKey", "0x0", "Hedera Private Key")
 	accountID := flag.String("accountId", "0.0", "Hedera Account ID")
 	network := flag.String("network", "", "Hedera Network Type")
+	members := flag.Int("members", 1, "The count of the members")
 	flag.Parse()
-
 	if *privateKey == "0x0" {
 		panic("Private key was not provided")
 	}
 	if *accountID == "0.0" {
 		panic("Account id was not provided")
 	}
+
 	fmt.Println("-----------Start-----------")
 	client := initClient(*privateKey, *accountID, *network)
-	privKey1, err := cryptoCreate(client)
-	if err != nil {
-		panic(err)
-	}
-	privKey2, err := cryptoCreate(client)
-	if err != nil {
-		panic(err)
-	}
-	privKey3, err := cryptoCreate(client)
-	if err != nil {
-		panic(err)
+
+	var memberKeys []hedera.PrivateKey
+	for i := 0; i < *members; i++ {
+		privKey, err := cryptoCreate(client)
+		if err != nil {
+			panic(err)
+		}
+		memberKeys = append(memberKeys, privKey)
 	}
+
+	fmt.Println("Private keys array:", memberKeys)
+
 	topicKey := hedera.KeyListWithThreshold(1)
-	topicKey = topicKey.
-		Add(privKey1.PublicKey()).
-		Add(privKey2.PublicKey()).
-		Add(privKey3.PublicKey())
+	for i := 0; i < *members; i++ {
+		topicKey.Add(memberKeys[i].PublicKey())
+	}
+
 	txID, err := hedera.NewTopicCreateTransaction().
 		SetSubmitKey(topicKey).
 		Execute(client)
 	if err != nil {
 		panic(err)
 	}
+
 	topicReceipt, err := txID.GetReceipt(client)
 	if err != nil {
 		panic(err)
 	}
+
 	fmt.Printf("TopicID: %v\n", topicReceipt.TopicID)
 	fmt.Println("--------------------------")
+
 	custodialKey := hedera.KeyListWithThreshold(3)
```

Comment:
This will always have a threshold `3`. You can use the input `members` parameter